streamer/pulp/streamer/server.py

```diff
@@ -95,16 +95,15 @@ def download_failed(self, report):
         :param report: The download report for this request.
         :type  report: nectar.report.DownloadReport
         """
-        error_report = report.error_report.copy()
-        error_report['url'] = report.url
-        logger.info(_('Download of %(url)s failed: HTTP %(response_code)s '
-                      '%(response_msg)s' % error_report))
```

Comment:
Should we retain this log statement, but move it into the if block below so we can be sure there is a response_code?